website/static/js/registrationUtils.js

```diff
@@ -463,6 +513,36 @@ RegistrationEditor.prototype.lastSaved = function() {
         return 'never';
     }
 };
+RegistrationEditor.prototype.viewComments = function() {
+	var self = this;
+
+	var comments = self.currentQuestion().comments();
```

Comment:
Please use 2 spaces for indents in JS:
https://github.com/felixge/node-style-guide#2-spaces-for-indention
https://github.com/felixge/node-style-guide#2-spaces-for-indention